src/encoded/static/components/browse/PublicationSearchView.js

```diff
@@ -0,0 +1,260 @@
+                </span>
+            );
+        }
+    },
+    "number_of_experiment_sets" : {
+        "widthMap" : { "sm" : 50, "md" : 50, "lg" : 50 },
+        "render" : function(result, columnDefinition, props, termTransformFxn){
+            const { number_of_experiment_sets: numSets = null } = result;
+            if (numSets === null) return null;
+            return (
+                <span className="value">
+                    { valueTransforms.decorateNumberWithCommas(numSets) }
+                </span>
+            );
+        }
+    }
+});
+
+/**
+ * This is memoized into a couple of different functions - 1 per column (common colWidth between all cells in same col)
+ * Make sure to keep this in sync with CSS rules (line height, etc.)
+ */
+function maxCharsToShowPerLine(colWidth){
+    const charWidth = 6; // Rough upper estimate (excl "i", "l", ".", etc.) for this font/size/weight
+    // -16px for table cell padding (8px left + right)
+    const textAreaWidth = colWidth - 60 - 16; // Account for line splitting on spaces (not in middle of words)
+    return Math.floor(textAreaWidth / charWidth);
+}
+
+
+class PublicationSearchResultTitle extends React.PureComponent {
+
+    static maxCharsToShowPerLine = memoize(maxCharsToShowPerLine);
+
+    static buildAuthorsElement(maxCharLen, authors = []){
+        const initAuthorsLen = authors.length;
+        if (initAuthorsLen === 0) return null;
+        const initAuthors = authors.slice(0);
+        const authorsToKeep = [];
+        let charCount = 0;
+        while (charCount < maxCharLen && initAuthors.length){
+            const author = initAuthors.shift();
+            const authorLen = author.length;
+            charCount += authorLen;
+            if (charCount < maxCharLen) {
+                authorsToKeep.push(author);
+            } else {
+                break;
+            }
+        }
+        const authorsToKeepLen = authorsToKeep.length;
+        if (initAuthorsLen === authorsToKeepLen) {
+            return authorsToKeep.join(" &bull; ");
+        } else {
+            return authorsToKeep.join(", ") + `, et al.`;
+        }
+    }
+
+    constructor(props){
+        super(props);
+        this.onClickTrack = this.onClickTrack.bind(this);
+        this.memoized = {
+            buildAuthorsString : memoize(PublicationSearchResultTitle.buildAuthorsString)
+        };
+    }
+
+    onClickTrack(evt){
+        const { result, href, navigate = spcNavigate } = this.props;
+        evt.preventDefault();
+        evt.stopPropagation();
+        analytics.productClick(result, {
+            'list'      : analytics.hrefToListName(href),
+            'position'  : rowNumber + 1
+        }, function(){
+            navigate(object.itemUtil.atId(result));
+        });
+        return false;
+    }
+
+    render(){
+        const { result, detailOpen, toggleDetailOpen, rowNumber, href, navigate = spcNavigate, width: colWidth } = this.props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['navigate' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4551745)